Resources/Private/Templates/Haiku/List.html

```diff
@@ -0,0 +1,13 @@
+<html data-namespace-typo3-fluid="true" xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers">
+
+<f:layout name="Default"/>
```

Comment:
You use layout but haven't defined one?

Reply:
I am using the Layout which has already been there

Reply:
Ah, it's like a content element, so that one is used. I am too long out of this stuff.